demo/src/main/java/com/afoxplus/uikit/demo/components/compose/ComposePreviewActivity.kt

```diff
@@ -111,6 +112,38 @@ class ComposePreviewActivity : ComponentActivity() {
                         ) {
                             println("Here is the data: $it")
                         }
+                        Spacer(modifier = Modifier.height(UIKitTheme.spacing.spacing10))
```

Comment:
Mr. @JhPetter avoid add previews in Compose Preview Activity. The best way...You should create an activity and add it to the MainActivity stack